pipeline/00-ingest.R

```diff
@@ -52,9 +53,11 @@ training_data <- dbGetQuery(
   WHERE condo.year
       BETWEEN '{params$input$min_sale_year}'
       AND '{params$input$max_sale_year}'
+  AND sale.deed_type IN ('01', '02', '05')
   AND NOT sale.sale_filter_same_sale_within_365
   AND NOT sale.sale_filter_less_than_10k
   AND NOT sale.sale_filter_deed_type
+  AND NOT sale.is_multisale
```

Comment:
I think we decided this shouldn't be here.

Reply:
Fixed by 8218098